app/src/main/res/layout/activity_quiz.xml

```diff
@@ -8,10 +8,10 @@
     >
 
     <TextView
+        android:id="@+id/question"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:padding="24dp"
-        android:text="@string/question_text"
```

Comment:
you shouldn't remove this line, use the tools namespace, `tools:text="@string/question_text"`. The main reason of this is that the layout will be fine in the preview